### PR TITLE
Rename ukrainian cities to proper spelling

### PR DESCRIPTION
**Pull request by bgr11n**
Please, rename ukrainian cities to use up-to-date spelling, examples from wiki:
- [Kyiv](https://en.wikipedia.org/wiki/Kyiv)
- [Zaporizhzhia](https://en.wikipedia.org/wiki/Zaporizhzhia)
- [Uzhhorod](https://en.wikipedia.org/wiki/Uzhhorod)

Also maybe it make sense to have separately some id or slug field that should be persistent and name that may change?

### CODE REVIEW

**Comment by tkuchiki** [comment]
@bgr11n Thank you for creating a PR, and I'm sorry for the late response.
I will check it later.

> Also maybe it make sense to have separately some id or slug field that should be persistent and name that may change?

You're right, so I will improve it later, thank you!

File: timezones.go [hunk 1]
```diff
@@ -140,16 +140,16 @@ var timezones = map[string][]string{
 		"Europe/Chisinau",
 		"Europe/Helsinki",
 		"Europe/Kaliningrad",
-		"Europe/Kiev",
+		"Europe/Kyiv",
```
Comment (tkuchiki) [inline timezones.go:143]:
```suggestion
		"Europe/Kiev",
		"Europe/Kyiv",
```

File: timezones.go [hunk 2]
```diff
@@ -140,16 +140,16 @@ var timezones = map[string][]string{
...
 		"Europe/Mariehamn",
 		"Europe/Nicosia",
 		"Europe/Riga",
 		"Europe/Sofia",
 		"Europe/Tallinn",
 		"Europe/Tiraspol",
-		"Europe/Uzhgorod",
+		"Europe/Uzhhorod",
```
Comment (tkuchiki) [inline timezones.go:150]:
```suggestion
		"Europe/Uzhgorod",
		"Europe/Uzhhorod",
```

File: timezones.go [hunk 3]
```diff
@@ -140,16 +140,16 @@ var timezones = map[string][]string{
...
 		"Europe/Vilnius",
-		"Europe/Zaporozhye",
+		"Europe/Zaporizhzhia",
```
Comment (tkuchiki) [inline timezones.go:152]:
```suggestion
		"Europe/Zaporozhye",
		"Europe/Zaporizhzhia",
```

File: timezones.go [hunk 4]
```diff
@@ -168,16 +168,16 @@ var timezones = map[string][]string{
 		"Europe/Chisinau",
 		"Europe/Helsinki",
 		"Europe/Kaliningrad",
-		"Europe/Kiev",
+		"Europe/Kyiv",
```
Comment (tkuchiki) [inline timezones.go:171]:
```suggestion
		"Europe/Kiev",
		"Europe/Kyiv",
```

File: timezones.go [hunk 5]
```diff
@@ -168,16 +168,16 @@ var timezones = map[string][]string{
...
 		"Europe/Mariehamn",
 		"Europe/Nicosia",
 		"Europe/Riga",
 		"Europe/Sofia",
 		"Europe/Tallinn",
 		"Europe/Tiraspol",
-		"Europe/Uzhgorod",
+		"Europe/Uzhhorod",
```
Comment (tkuchiki) [inline timezones.go:178]:
```suggestion
		"Europe/Uzhgorod",
		"Europe/Uzhhorod",
```

File: timezones.go [hunk 6]
```diff
@@ -168,16 +168,16 @@ var timezones = map[string][]string{
...
 		"Europe/Vilnius",
-		"Europe/Zaporozhye",
+		"Europe/Zaporizhzhia",
```
Comment (tkuchiki) [inline timezones.go:180]:
```suggestion
		"Europe/Zaporozhye",
		"Europe/Zaporizhzhia",
```

File: tz_infos.go [hunk 7]
```diff
@@ -7153,7 +7153,7 @@ var tzInfos = map[string]*TzInfo{
 		linkTo:             "",
 		lastDST:            2010,
 	},
-	"Europe/Kiev": &TzInfo{
```
Comment (tkuchiki) [inline tz_infos.go:7156]:
```go
	"Europe/Kiev": &TzInfo{
		longGeneric:        "Eastern European Time",
		longStandard:       "Eastern European Standard Time",
		longDaylight:       "Eastern European Summer Time",
		shortGeneric:       "EET",
		shortStandard:      "EET",
		shortDaylight:      "EEST",
		standardOffset:     7200,
		daylightOffset:     10800,
		standardOffsetHHMM: "+02:00",
		daylightOffsetHHMM: "+03:00",
		countryCode:        "UA",
		isDeprecated:       false,
		linkTo:             "",
		lastDST:            2037,
	},
```

File: tz_infos.go [hunk 8]
```diff
@@ -7649,7 +7649,7 @@ var tzInfos = map[string]*TzInfo{
 		linkTo:             "",
 		lastDST:            2010,
 	},
-	"Europe/Uzhgorod": &TzInfo{
```
Comment (tkuchiki) [inline tz_infos.go:7652]:
```go
	"Europe/Uzhgorod": &TzInfo{
		longGeneric:        "Eastern European Time",
		longStandard:       "Eastern European Standard Time",
		longDaylight:       "Eastern European Summer Time",
		shortGeneric:       "EET",
		shortStandard:      "EET",
		shortDaylight:      "EEST",
		standardOffset:     7200,
		daylightOffset:     10800,
		standardOffsetHHMM: "+02:00",
		daylightOffsetHHMM: "+03:00",
		countryCode:        "UA",
		isDeprecated:       false,
		linkTo:             "",
		lastDST:            2037,
	},
```

File: tz_infos.go [hunk 9]
```diff
@@ -7777,7 +7777,7 @@ var tzInfos = map[string]*TzInfo{
 		linkTo:             "Europe/Belgrade",
 		lastDST:            2037,
 	},
-	"Europe/Zaporozhye": &TzInfo{
```
Comment (tkuchiki) [inline tz_infos.go:7780]:
```go
	"Europe/Zaporozhye": &TzInfo{
		longGeneric:        "Eastern European Time",
		longStandard:       "Eastern European Standard Time",
		longDaylight:       "Eastern European Summer Time",
		shortGeneric:       "EET",
		shortStandard:      "EET",
		shortDaylight:      "EEST",
		standardOffset:     7200,
		daylightOffset:     10800,
		standardOffsetHHMM: "+02:00",
		daylightOffsetHHMM: "+03:00",
		countryCode:        "UA",
		isDeprecated:       false,
		linkTo:             "",
		lastDST:            2037,
	},
```

**Review by tkuchiki — CHANGES_REQUESTED** [review]
Could you please add it rather than rename it for backward compatibility?

**Comment by bgr11n** [comment]
@tkuchiki But in that case there will be duplicates?

**Comment by tkuchiki** [comment]
Yes. In that case, do you have any concerns?

**Comment by bgr11n** [comment]
@tkuchiki No, just wanted to clarify that it is expected.

**Review by tkuchiki — APPROVED** [review]
LGTM, thank you!

**Pull request merged**